hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/DeleteTableProcedure.java

```diff
@@ -133,7 +137,13 @@ protected Flow executeFromState(final MasterProcedureEnv env, DeleteTableState s
       if (isRollbackSupported(state)) {
         setFailure("master-delete-table", e);
       } else {
-        LOG.warn("Retriable error trying to delete table=" + getTableName() + " state=" + state, e);
+        if (retryCounter == null) {
+          retryCounter = ProcedureUtil.createRetryCounter(env.getMasterConfiguration());
+        }
+        long sleepTime = retryCounter.getBackoffTimeAndIncrementAttempts();
+        LOG.warn("Retriable error trying to delete table=" + getTableName() + " state=" + state
+          + ", Will sleep " + sleepTime / 1000 + " secs and try again later", e);
+        Threads.sleep(sleepTime);
```

Comment:
Better not sleep directly here as it will block e PEWorker. You can call suspend method from the parent Procedure class to suspend the procedure for a while.

And we also need to reset the retryCounter if there is no problem after retrying.

Reply:
thanks for reviwing
1. i have some questions...
you suggest call suspend method from the parent Procedure? 
but CreateTableProcedure did not have a parent proc, or you mean we need add a suspent method to
AbstractStateMachineTableProcedure?
For the old logic, althouth TRSP could suspent, but CreateTableProcedure could not..

3.yeah, we need reset the retryCounter...

Reply:
The suspend method is in the Procedure class, which is the parent class for all procedures.

Reply:
Check out https://github.com/apache/hbase/issues/5534 for a recent example of doing this pausing